mock/py/mockbuild/package_manager.py

```diff
@@ -328,11 +328,33 @@ def copy_certs(self):
                 # when mock is not run on Fedora or EL
                 self.buildroot.root_log.debug("ca bundle not found on host")
 
+    @traceLog()
+    def copy_extra_certs(self):
+        extra_certs = self.config['ssl_extra_certs']
+        if extra_certs:
+            self.buildroot.root_log.debug('copying extra certificates into chroot')
+            for cert_src, cert_dest in zip(extra_certs[::2], extra_certs[1::2]):
+                host_cert_src = os.path.realpath(cert_src)
+                chroot_cert_dest = self.buildroot.make_chroot_path(cert_dest)
+                chroot_cert_dir = os.path.dirname(chroot_cert_dest)
+                try:
+                    os.makedirs(chroot_cert_dir)
```

Comment:
Can you use file_util.mkdirIfAbsent() ? Then you can remove that try/except block.

Reply:
Done.